Resources/Documentation/linux_installation.md

```diff
@@ -23,10 +23,13 @@ There are many possible ways to install the various dependencies of Vrecord on L
   - `sudo apt-get install xmlstarlet`
   - `sudo apt-get install mkvtoolnix`
   - `sudo apt-get install mediaconch`
+* Install the following dependencies for enabling DV capture in vrecord's FFmpeg build:
+  - `sudo apt-get install libiec61883-dev`
+  - `sudo apt-get install libraw1394-dev`
+  - `sudo apt-get install libavc1394-dev`
 * If `make` and `gcc` are not already installed, install them with
   - `sudo apt-get install gcc`
   - `sudo apt-get install make`
-
```

Comment:
Nit: Is there a reason to delete this line? (To me, it makes the code easier to read.)

Reply:
No real reason - I didn't like it in the moment, but looking back at the rest of the doc, would have to remove all the other spaces for consistency, so I'll add it back in!

Reply:
@privatezero Using sed `/^$/d` would remove all the others… ;-)